metis-harvesting/src/main/java/eu/europeana/metis/harvesting/oaipmh/OaiRecordHeader.java

```diff
@@ -37,7 +37,7 @@ public OaiRecordHeader(String oaiIdentifier, boolean isDeleted, Instant datestam
    */
   static OaiRecordHeader convert(Header header) {
     return new OaiRecordHeader(header.getIdentifier(), header.isDeleted(),
-            Optional.ofNullable(header.getDatestamp()).map(Date::toInstant).orElse(null));
+            Optional.ofNullable(header.getDatestamp()).orElse(null));
```

Comment:
This chain is unnecessary? Just use header.getDatestamp()

Reply:
oops. true. done.